crates/goose/src/providers/formats/bedrock.rs

```diff
@@ -62,6 +62,14 @@ pub fn to_bedrock_message_content(content: &MessageContent) -> Result<bedrock::C
                 Ok(content) => Some(
                     content
                         .iter()
+                        // Filter out content items that have User in their audience
+                        .filter(|c| {
+                            if let Some(audience) = c.audience() {
+                                !audience.contains(&Role::User)
+                            } else {
+                                true // Include if no audience annotation
+                            }
+                        })
```

Comment:
you could also write this as:
```rust
.filter(|c| c.audience().map_or(true, |audience| !audience.contains(&Role::User)))
```
`map_or` would default to `true` in the `None` case, and run the filter otherwise